snapcraft/main.py

```diff
@@ -15,131 +15,82 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import argparse
+"""
+snapcraft
+
+Usage:
+  snapcraft [--version | --help] [options] COMMAND [ARGS ...]
+
+Options:
+  -h --help        show this help message and exit
+  -v --version     show program version and exit
+  -V --verbose     print additional information about command execution
+
+The available commands are:
+  list-parts   List available parts which are like “source packages” for snaps.
+  list-plugins List the available plugins that handle different types of part.
+  init         Initialize a snapcraft project.
+  add-part     Add a part to your snapcraft.yaml, interactively presenting
+               options.
+  help         Obtain help for a certain plugin or topic
+
+The available lifecycle commands are:
+  clean        Remove content - download, build or install artifacts.
```

Comment:
This is confusing, because this can mean that clean downloads, builds or installs artifacts.
Maybe: remove downloaded, built or installed artifacts.